processing/src/main/java/io/druid/segment/IndexMaker.java

```diff
@@ -1220,7 +1282,8 @@ private static void makeMetricColumn(
       final String metric,
       final Map<String, ValueType> valueTypes,
       final Map<String, String> metricTypeNames,
-      final int rowCount
+      final int rowCount,
+      final CompressedObjectStrategy.CompressionStrategy compression
```

Comment:
Can we call this `compressionStrategy` instead?  I've had to ask myself a couple times, "what's a compression again?"